### DISCUSSION TITLE
Batch files for creating and verifying PAR files for directories!

### DISCUSSION THREAD

**Discussion by jonathanmc** [Yutaka-Sawada/MultiPar#63] [Show and tell]
Great utility. Love it.

Here is my little contribution: https://github.com/jonathanmc/multipar_scripts

This repo has batch file scripts that make PAR files for directories or files selected in Explorer. You select, then use SEND TO to select the batch file. 

I hope it is useful!

**Comment by JohnLGalt** [comment]
Nice job!
